### ISSUE TITLE
RFC: Consider weaponising search boxes for keyboard jockeys

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern#1722]
https://forum.textpattern.com/viewtopic.php?id=51585

Currently, our search process for articles (files, images, and so on) is quite click-heavy. If I want to search for articles with a status of Draft, I can:

- select the dropdown header
- select the ‘Toggle all/none’ checkbox
- select the ‘Status’ checkbox
- select the ‘Search articles’ field
- type Draft
- press Enter or select the magnifying glass

We should consider improving our search process to allow for a more direct search. Considerations:

* search for content by attribute status (e.g. `status:draft`)
* boolean stuff, and the correct construction of (e.g. 'status is draft AND author is Alice')
* permit i18n (with or without caps) for a smoother ride (e.g. `status:Entwurf`, `estado:borrador`)
* if this extends to dates, a range should be searchable
* choice of delimiter (e.g. `::`), perhaps specified in an advanced preference

Much more on this here from @Bloke: https://forum.textpattern.com/viewtopic.php?pid=331195#p331195
